bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/BehaviorConverter.java

```diff
@@ -164,28 +203,33 @@ private Term createTerm(String token) {
 
     }
 
-    private String termToString(Term term, boolean isNested) {
+    private String termToString(Term term, List<String> flowNames, boolean isNested) {
         if (term instanceof LabelReference labelReference) {
-            return labelReference.getLabel()
-                    .getEntityName();
+        	StringBuilder builder = new StringBuilder();
+        	for (int i = 0; i < flowNames.size(); i++) {
+        		if (i != 0) builder.append("_");
```

Comment:
Shoulnd the underscore be a `;`or even better use the DELIMITER constant from the DFDConverter

Reply:
Yes